CPT331.iOS/CPT331.iOS/EventManager.swift

```diff
@@ -62,14 +74,16 @@ class EventManager {
                     let name = event["name"].string
```

Comment:
One advantage of using an enum is you get a (_small_) degree of compile time safety.

e.g. It's a lot harder to mix up "Point" instead of "point" when the enum is compiled.

I'm happy with either fix... using enums or removing the unused enum types.